cmd/registry/core/print.go

```diff
@@ -61,7 +61,7 @@ func PrintSpecDetail(message *rpc.ApiSpec) {
 
 func PrintSpecContents(message *rpc.ApiSpec) {
 	contents := message.GetContents()
-	if strings.HasSuffix(message.GetMimeType(), "+gzip") {
+	if strings.Contains(message.GetMimeType(), "+gzip") {
```

Comment:
This is another fix to the way we recognize content types (this can be improved further in the future)